src/clj/game/core/cards.clj

```diff
@@ -52,7 +52,7 @@
       (update-hosted! state side card)
       (let [z (cons (to-keyword (or (get-scoring-owner state card) (:side card))) zone)
             [head tail] (split-with #(not= (:cid %) cid) (get-in @state z))]
-        (when-not (empty? tail)
+        (when (seq tail)
```

Comment:
See @nealterrell regarding a discussion why we choose to use not-empty instead of `seq`...

Reply:
What's up with this?